config.c

```diff
@@ -740,6 +777,7 @@ int parse_config_arguments(struct mako_config *config, int argc, char **argv) {
 		{"max-visible", required_argument, 0, 0},
 		{"max-history", required_argument, 0, 0},
 		{"history", required_argument, 0, 0},
+		{"surface", required_argument, 0, 0},
```

Comment:
This is a leftover from previous code since the `surface` option has been removed, correct?